### ISSUE TITLE
Error after installing the plugin

### ISSUE THREAD

**Issue by n3bojs4** [ffddorf/netbox-vault-secrets#7]
Hello,
I manually installed the plugin, following the netbox doc : https://netbox.readthedocs.io/en/stable/plugins/
But after that netbox can't start. This is the log below. Can you understand what's going wrong ?

Netbox v3.0.2

```
Traceback (most recent call last):
  File "/data/netbox-3.0.2/netbox/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 363, in execute
    settings.INSTALLED_APPS
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/data/netbox-3.0.2/netbox/netbox/settings.py", line 584, in <module>
    plugin = importlib.import_module(plugin_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/netbox_vault_secrets-0.0.1-py3.9.egg/netbox_vault_secrets/__init__.py", line 7, in <module>
    class VaultSecretsConfig(PluginConfig):
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/netbox_vault_secrets-0.0.1-py3.9.egg/netbox_vault_secrets/__init__.py", line 8, in VaultSecretsConfig
    name = pkg_info.name
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/pkg_resources/__init__.py", line 2816, in __getattr__
    return getattr(self._provider, attr)
AttributeError: 'PathMetadata' object has no attribute 'name'
(venv) root@svc-ces8-lab1:/opt/netbox/netbox# vi netbox/configuration.py
(venv) root@svc-ces8-lab1:/opt/netbox/netbox# python manage.py collectstatic
/data/netbox-3.0.2/netbox/netbox/settings.py:50: UserWarning: The CACHE_TIMEOUT configuration parameter was removed in v3.0.0 and no longer has any effect.
  warnings.warn(
Traceback (most recent call last):
  File "/data/netbox-3.0.2/netbox/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 363, in execute
    settings.INSTALLED_APPS
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/data/netbox-3.0.2/netbox/netbox/settings.py", line 584, in <module>
    plugin = importlib.import_module(plugin_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/netbox_vault_secrets-0.0.1-py3.9.egg/netbox_vault_secrets/__init__.py", line 7, in <module>
    class VaultSecretsConfig(PluginConfig):
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/netbox_vault_secrets-0.0.1-py3.9.egg/netbox_vault_secrets/__init__.py", line 8, in VaultSecretsConfig
    name = pkg_info.name
  File "/data/netbox-3.0.2/venv/lib/python3.9/site-packages/pkg_resources/__init__.py", line 2816, in __getattr__
    return getattr(self._provider, attr)
AttributeError: 'PathMetadata' object has no attribute 'name'
```

**Comment by mraerino** [comment]
thanks for the report, will try to solve this

**Comment by mraerino** [comment]
the supposed fix in #8 has a release artifact: https://github.com/ffddorf/netbox-vault-secrets/actions/runs/1589481471
can you test if that fixes the issue?

**Comment by syreel** [comment]
This patch fixed this issue for me

**Comment by n3bojs4** [comment]
> the supposed fix in #8 has a release artifact: https://github.com/ffddorf/netbox-vault-secrets/actions/runs/1589481471 can you test if that fixes the issue?

Hello, i manually fixed the 2 files (setup.py and __init__.py) and it fixes the issue.
For info, the patch is not in the release archive it's why i was confused.
Thanks.
